app/views/landing/index.html.erb

```diff
@@ -0,0 +1,257 @@
+                <h3 class="font-bold text-3xl mb-5">Alex</h3>
+                <p class="italic text-[rgba(255,255,255,0.678)]">
+                  "I'm 100% convinced that this is how we cultivate the next generation of truly powerful software engineers."
+                </p> 
+              </div>
+            </div>
+          </div>
+        </div>
+
+      </div>
+    </div>
+
+    <hr class="border-t-1 border-[rgba(255,255,255,0.051)]">
+    <!-- "The Pain" -->
+    <div class="py-12 mb-5">
+      <div class="container mx-auto px-4">
+        <h2 class="text-4xl mb-8 text-center font-extrabold ">Being a junior dev is lonely</h2>
+
+        <div class="flex justify-center mb-5">
+          <div class="flex justify-center gap-1 flex-col md:flex-row">
+            <div class="p-2 w-full md:w-1/2  mx-auto grid justify-items-end ">
+              <%= image_tag "welcome/the_pain.jpeg", class:" rounded object-cover ", alt:"Description of Image" %>
+            </div>
+            <div class="p-2 w-full md:w-1/2 max-w-prose mx-auto flex-1 leading-loose text-base font-medium">
+              <p class="text-[rgba(255,255,255,0.678)]">
+                <span class="p">Joining a developer community helps you meet peers, get advice, get unstuck, practice tricky situations, and grow your network.</span> <br>
+                <span class="p">Mentorship and collaboration are also very hard to find when you’re solo.</span> <br>
+                <span class="p">That’s what The Agency of Learning is for.</span> <br>
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <hr class="border-t-1 border-[rgba(255,255,255,0.051)]">
+    <!--  Members Benefits -->
+    <div class="py-12 mb-5">
+      <div class="container mx-auto px-4">
+        <h2 class="text-4xl mb-8 text-center font-extrabold ">Member Benefits</h2>
+
+        <div class="flex justify-center mb-5 leading-loose text-base font-medium">
+          <div class="flex justify-center gap-1 flex-col md:flex-row">
+            <div class="p-2 w-full md:w-1/2  mx-auto grid justify-items-end ">
+              <%= image_tag "welcome/real_code.jpeg", class:" rounded object-cover ", alt:"Description of Image" %>
+            </div>
+
+            <div class="p-2 w-full md:w-1/2 max-w-prose mx-auto flex-1">
+              <strong>Real code, real peers</strong>
+              <p class="text-[rgba(255,255,255,0.678)]">Work together on software projects on a team with other developers. Get feedback on your code from experienced engineers. Talk to users.</p>
+            </div>
+
+          </div>
+        </div>
+        <div class="flex justify-center mb-5 leading-loose text-base font-medium">
+          <div class="flex justify-center gap-1 flex-col md:flex-row">
+            <div class="p-2 w-full md:w-1/2 max-w-prose mx-auto flex-1">
+              <strong>Mastermind Community</strong>
```

Comment:
So I realize this is how it is on the current main site, but it feels weird to me how on mobile, you get 2 paragraphs together, then two images.
![Selection_243](https://github.com/agency-of-learning/PairApp/assets/88392688/b144b931-d0ff-4e08-8ac7-eea04b26e271)
![Selection_244](https://github.com/agency-of-learning/PairApp/assets/88392688/16aa522f-082a-48a8-b0c6-bd79c91e2c0a)

You can fix this but still keep the staggered layout on desktop by changing `flex-col` to `flex-col-reverse` on line 174. And you'd want to do this same thing further down the page where this also occurs.